deploy/health_check/requirments.txt

```diff
@@ -0,0 +1,3 @@
+Flask==0.12.2
+Flask-Cors==3.0.6
+requests==2.18.4
```

Comment:
- i would use pipenv instead of pip requirements.txt and read how to deploy it onto production